eahub/templates/base_bs5.html

```diff
@@ -21,3 +21,18 @@
     <link  rel="stylesheet" href="{% static 'dist/global_bs5.css' %}">
   {% endif %}
 {% endblock static %}
+
+{% block messages %}
+
+{% if messages %}
+<div class="container mt-2 w-75">
+  {% for message in messages %}
+  <div class="alert {% for tag in message.tags %} alert-{{ tag }} {% endfor %} alert-dismissible fade show" role="alert">
```

Comment:
* Testing this, I've noticed that the loop is incorrect here. ```message.tags``` is not a set of values, but just one (e.g., "success"), so the loop ends up looping over letters. The result of this is that the css is messed up and you actually have no styling on the message.

You can just do ```alert-{{message.tags}}``` here.

* this is missing the hub-{{message.tags}} (the bootstrap default color doesn't match the hub colour pallette)